### PR TITLE
Bonuses and stipends

### PR DESCRIPTION
**Pull request by cristinabaldor**
# Pull Request

## Merge Pre-Check

- [x] Create a Pull Request task on our
      [Data Warehouse](https://app.asana.com/0/35737884613515/1204991342481434) Asana project
- [x] <kbd>Format</kbd> has been run on all requested files
- [x] `dbt build` successfully run for the entire project[^1]

[^1]:
    If only requesting 1 file, it's acceptable to run build only for that and its downstream
    dependencies.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205770554892480

### CODE REVIEW

File: src/dbt/kipptaf/models/extracts/google/sheets/rpt_gsheets__stipends_app_roster.sql [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                'Director',
+                'School Leader',
+                'School Leader in Residence'
+            )
+            and assignment_status != 'Terminated'
+    ),
+
+    roster as (
+        select
+            sr.employee_number,
```
Comment (cbini) [inline src/dbt/kipptaf/models/extracts/google/sheets/rpt_gsheets__stipends_app_roster.sql:43]: group cols by source table and order from simple (no changes) to complex (case statements)

**Review by cbini — CHANGES_REQUESTED** [review]

**Comment by cristinabaldor** [comment]
wasn't sure about the second one -- did you just mean to move down the campus_crosswalk item to the end?

**Comment by cbini** [comment]
> wasn't sure about the second one -- did you just mean to move down the campus_crosswalk item to the end?

here's an example of what I'm talking about: [group the columns from each table in blocks, starting from simple columns to calc etc](https://docs.sqlfluff.com/en/stable/rules.html#rule-ST06)

File: src/dbt/kipptaf/models/extracts/google/sheets/rpt_gsheets__stipends_app_roster.sql [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                else 'Regional'
+            end as route,
+            coalesce(cc.name, sr.home_work_location_name) as campus,
+        from {{ ref("base_people__staff_roster") }} as sr
+        left join
+            {{ ref("stg_people__campus_crosswalk") }} as cc
+            on sr.home_work_location_name = cc.location_name
+        where
+            sr.worker_termination_date is null
+            or sr.worker_termination_date > '2023-07-01'
```
Comment (cbini) [inline src/dbt/kipptaf/models/extracts/google/sheets/rpt_gsheets__stipends_app_roster.sql:87]: should this be dyanamic for the current year?

Comment (cristinabaldor) [reply src/dbt/kipptaf/models/extracts/google/sheets/rpt_gsheets__stipends_app_roster.sql:87]: i'm going to have to change it every quarter, we can leave as is for now

**Review by cbini — APPROVED** [review]

**Pull request merged**